Purchases/Purchasing/RCStoreKitWrapper.m

```diff
@@ -66,6 +66,16 @@ - (void)finishTransaction:(SKPaymentTransaction *)transaction
     [self.paymentQueue finishTransaction:transaction];
 }
 
+- (void)presentCodeRedemptionSheet {
+    #if TARGET_OS_IOS
+    if (@available(iOS 14.0, *)) {
+        [self.paymentQueue presentCodeRedemptionSheet];
+    } else {
+        // Fallback on earlier versions
```

Comment:
let's log a warning here saying that `presentCodeRedemptionSheet` was attempted, but isn't available on this device, for debugging purposes.